README.md

```diff
@@ -389,6 +418,12 @@ recovery code is the new recovery code you must show to the user from him to sav
 transaction.recover({ recoveryCode: recoveryCode });
 ```
 
+#### transaction.serialize() 
+The .serialize() method creates a regular javascript Object that should remind opaque 
```

Comment:
a "plain" object, or something better? A javascript object might have functions and other things in it. Maybe a plain JSON Object?